treeherder/autoclassify/detect_intermittents.py

```diff
@@ -1,8 +1,8 @@
 import logging
```

Comment:
I don't think it's "different" to the code we're running in production; it's just preffed off. That seems fine to me. Nevertheless, I removed it, but it at least seems pretty weird to bundle it with these unrelated changes.